app/controllers/api/api_controller.rb

```diff
@@ -101,6 +99,122 @@ def dashboard
     end
   end
 
+  # Get Team & Team Role & Permissions info for the Mobile App.
+  def mobile_basic
```

Comment:
Here's our mobile_basic query. We simply get the teams, user's team, and permissions.